dags/atd_knack_artbox_signals.py

```diff
@@ -17,7 +17,7 @@
     "email_on_failure": False,
     "email_on_retry": False,
     "retries": 0,
-    "retry_delay": duration(minutes=5),
+    "execution_timeout": duration(minutes=30),
```

Comment:
is this one 30 instead of its previous 5 min, because its a slower ETL?

Reply:
nope—thank you!